### ISSUE TITLE
Writeable and pushable streamed repos

### ISSUE THREAD

**Issue by erinys** [xetdata/xet-tools#47]
From Snehesh via Discord:

TLDR
I was wondering if there is a workflow to have the best of both worlds of stream and xethub repo as a writable/ pushable stream repo? Akin to Xethub of google drive.

DETAILS
Hi I am exploring a use case that I could use your guidance on how to leverage Xethub. I am working on a large dataset with many studio recording. I was able to resolved the preview issue of the videos by manually also creating a preview folder with a short 10s video extracted at a slightly lower quality for each video. I was also able to stream clone the repo in a new folder. This was perfect. However, I believe the stream repo is readonly right? What I would ideally like to do is this:
1. Create a repo and push the data set part 1 (let's say dataset_p1 folder)
2. Once pushed, I do not need all the files that are push locally so I would like them esp specific folders to now be stream
3. But be able to add new data, for example now data set part 2 (dataset_p2 folder) and push them.

Ideally I would like to be able to do this at a file level but folder level is workable.

**Comment by erinys** [comment]
We do have a writeable mount option that is not fully tested and may be buggy and finicky, that would address this use case. Try using the `-w` flag: `git xet mount -w ...` and let us know how it works for you.

**Comment by sneheshs** [comment]
Thank you @erinys! Will give it a try and update here.

**Comment by allisonwhilden** [comment]
We are tracking this work internally with [PRO-83](https://linear.app/xethub/issue/PRO-83/surface-xet-ldfs-to-users). Closing this as a duplicate of that effort.
